policy/modules/system/systemd.if

```diff
@@ -519,6 +519,29 @@ template(`systemd_user_manager_dbus_chat',`
 	allow $1_systemd_t $2:dbus send_msg;
 ')
 
+######################################
+## <summary>
+##   Grant the specified systemd user manager
+##	 all the permissions necessary to run user
+##	 containers.
+## </summary>
+## <param name="prefix">
+##	<summary>
+##	Prefix for the user domain.
+##	</summary>
+## </param>
+#
+template(`systemd_user_use_containers',`
```

Comment:
I'm struggling to see why this is a systemd interfaces. Perhaps this should be held back until it has more access(?)

Reply:
I put this template in the systemd module since the access will be granted to systemd user managers on a per-role basis, e.g. `staff_systemd_t` can create containers but `user_systemd_t` cannot.

However, I think I'll drop this for now and revisit it once it's more complete as the implementation may change.